rust/common-ifc/src/context.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{PolicyError, Result};
+
+/// Environment a module is being evaluated in,
+/// ordered from least to most "private".
+#[derive(Ord, PartialOrd, Eq, PartialEq, Clone, Debug)]
+pub enum ModuleEnvironment {
+    /// Confidential compute environment.
+    Server,
+    /// On the client in a browser.
+    BrowserClient,
```

Comment:
I suspect we'll have begun experimenting with non-web browser clients before the year is over.

Is the plan to have multiple client types represented here?

If so, it would be nice to distinguish this as the hypertext web browser client.

Reply:
More placeholders than anything, and possibly something that needs to be defined outside the runtime -- while the wasm runtime will be used in a local browser context, not all wasm builds are local browser contexts. That is to say, a lot of TBD